src/components/AppBar/index.jsx

```diff
@@ -78,10 +81,22 @@ export default function AppBar(props) {
             Pulpito
           </RouterLink>
         </Typography>
+        <Button
+          component={RouterLink}
+          to="http://localhost:8082/login" 
+          color="inherit"
+          sx={{
+            bgcolor: "#b81f14", 
```

Comment:
```suggestion

```
Think color Red doesn't really fit well with our theme, we can remove it.

Reply:
Sure, any colour suggestions that we can use here?

Reply:
I think just deleting the line I suggested will do the trick :)